src/toTransform.js

```diff
@@ -73,8 +85,20 @@
     pathElement.setAttribute('d', offsetPath.input);
 
     var offsetDistanceLength = getOffsetDistanceLength(offsetDistance, pathElement.getTotalLength());
+    if (offsetDistanceLength < 0) {
+      offsetDistanceLength = (offsetDistanceLength % pathElement.getTotalLength()) + pathElement.getTotalLength();
+    }
+
+    if (closedLoop) {
+      offsetDistanceLength = offsetDistanceLength % pathElement.getTotalLength();
+    }
```

Comment:
No need to do this adjustment if it's within the total length, you can treat it the same as !closedLoop.